### PR TITLE
First approach to max-number validator

### PR DESCRIPTION
**Pull request by v-borrego**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,7 @@
 
 This is a [fonk](https://github.com/Lemoncode/fonk) microlibrary that brings validation capabilities to:
 
-// TODO: Update description and example.
-
-- Validate if a field of a form ....
+- Validate if a number field of a form is less (or optionally equal) than a maximum value
```
Comment (nasdan) [inline README.md]: lower instead of less

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@lemoncode/fonk-max-number-validator",
   "version": "0.1.0",
-  "description": "// TODO: Update description: This is a [fonk](https://github.com/Lemoncode/fonk) microlibrary that brings validation capabilities to validate if a field of a form ...",
+  "description": "Update description: This is a [fonk](https://github.com/Lemoncode/fonk) microlibrary that brings validation capabilities to validate if a number field of a form is less (or optionally equal) than a maximum value",
```
Comment (nasdan) [inline package.json]: lower

File: src/validator.spec.ts [hunk 3]
```diff
@@ -43,23 +45,219 @@ describe('fonk-max-number-validator specs', () => {
...
     expect(result).toEqual({
       succeeded: false,
-      message: 'other message',
-      type: '',
+      message: TEST_MESSAGE,
+      type: VALIDATOR_TYPE,
+    });
+  });
+
+  it('should return succeeded validation when it feeds value is less than max-value', () => {
```
Comment (nasdan) [inline src/validator.spec.ts]: lower

File: src/validator.spec.ts [hunk 4]
```diff
@@ -43,23 +45,219 @@ describe('fonk-max-number-validator specs', () => {
...
+
+    // Assert
+    expect(result).toEqual({
+      succeeded: true,
+      message: '',
+      type: VALIDATOR_TYPE,
+    });
+  });
+
+  it('should return failed validation when it feeds value is equals to max-value, within inclusive flag', () => {
```
Comment (nasdan) [inline src/validator.spec.ts:103]: within inclusive flag? or with inclusive flag equals false

File: src/validator.ts [hunk 5]
```diff
@@ -1,23 +1,42 @@
...
+import { CustomValidatorArgs } from './validator.model';
+import { buildCustomMessage, isDefined } from './validator.business';
 
-// TODO: Add validator type
-const VALIDATOR_TYPE = '';
+const VALIDATOR_TYPE = 'MAX_NUMBER';
 
-// TODO: Add default message
-let defaultMessage = '';
+let defaultMessage = 'The value must be less than or equal to {{maxValue}}';
```
Comment (nasdan) [inline src/validator.ts]: lower

Comment (v-borrego) [reply src/validator.ts]: Done

File: src/validator.spec.ts (hunk 4 above)
Comment (v-borrego) [reply src/validator.spec.ts:103]: Done

File: src/validator.spec.ts (hunk 3 above)
Comment (v-borrego) [reply src/validator.spec.ts]: Done

File: README.md (hunk 1 above)
Comment (v-borrego) [reply README.md]: Done

File: package.json (hunk 2 above)
Comment (v-borrego) [reply package.json]: Done

**Pull request merged**